src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/BackupConfiguration.kt

```diff
@@ -0,0 +1,49 @@
+package com.atlassian.performance.tools.jiraactions.api.page
+
+import com.atlassian.performance.tools.jiraactions.Patience
+import org.apache.logging.log4j.LogManager
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+
+internal class BackupConfiguration(
+    private val driver: WebDriver,
+    private val access: AdminAccess
+) {
+    private val logger = LogManager.getLogger(this::class.java)
+    private val backupLocator = By.xpath(
+        "//*[@id='tbl_services']" +
+            "//*[contains(text(), 'com.atlassian.jira.service.services.export.ExportService')]" +
+            "/ancestor::tr" +
+            "//*[contains(@class, 'operations-list')]" +
+            "//*[contains(text(), 'Delete')]"
+    )
+
+    fun disable(): BackupConfiguration {
+        val prompted = access.isPrompted()
+        if (prompted) {
+            access.gain()
+        }
+        if (isBackupServicePresent()) {
+            ensureSwitchIsOff()
```

Comment:
There's no switch on the page. This is copy-pasted from `RichTextEditorConfiguration`, which has a switch: 
![image](https://user-images.githubusercontent.com/1194704/62776968-5386df80-baac-11e9-825d-b668e8507009.png)